### DISCUSSION TITLE
Unable to change my email address on App

### DISCUSSION THREAD

**Discussion by agarw184** [community/community#55429] [Mobile]
I lost access to my previous email and I am trying to switch email address using the mobile app, but there is no such option. Could you please point me to it if there is one ? If not, request considering this addition ? 

Thanks! 

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.111.0 (97985743)
Device: iPhone 12 (iOS 16.4.1)
</details>

**Comment by ryanlua** [comment]
Hi @agarw184,

Unfortunately you cannot change emails via the GitHub Mobile app and are needed to use GitHub in the browser to change it. 

You should seek help from GitHub about your lost email by contacting them at [Help with your GitHub account](https://support.github.com/contact/account) to help recover your account.
